packages/react/.storybook/main.js

```diff
@@ -50,6 +50,12 @@ export default {
       titlePrefix: 'Live Demos/Index Explorer',
     },
   ],
+  features: { emotionAlias: false },
+  refs: {
+    '@chakra-ui/react': {
+      disable: true,
+    },
+  },
```

Comment:
Thanks for the question, I will add a comment.

That is because Charka comes with a native storybook automatically assigned to our storybook instance. This tells the storybook to not consider it.